maps.py

```diff
@@ -1,49 +1,47 @@
 from abc import ABC, abstractmethod
-from pathlib import Path
-from typing import Generator, Optional
-from urllib.parse import urlparse
+from typing import Generator, Optional, Type, Tuple
 
-from pygeotile.tile import Tile
+from darkgeotile import BaseTile, get_Tile
 
-from utils import ImageFormat
-from utils import get_random_tile
+from pyproj import Proj
 
 
 class Map(ABC):
-    def __init__(self, *args, **kwargs):
-        raise NotImplementedError
+    Tile: Type[BaseTile]
+
+    # Attributes for overwriting:
+    projection: Proj
+    projection_bounds: Optional[Tuple[float, float, float, float]] = None
```

Comment:
It is projected bounds not projection. See example here: http://spatialreference.org/ref/epsg/3322/